packages/app/src/heartbeatService.ts

```diff
@@ -80,43 +81,47 @@ export class HeartbeatServiceImpl implements HeartbeatService {
    * already logged, subsequent calls to this function in the same day will be ignored.
    */
   async triggerHeartbeat(): Promise<void> {
-    const platformLogger = this.container
-      .getProvider('platform-logger')
-      .getImmediate();
+    try {
+      const platformLogger = this.container
+        .getProvider('platform-logger')
+        .getImmediate();
 
-    // This is the "Firebase user agent" string from the platform logger
-    // service, not the browser user agent.
-    const agent = platformLogger.getPlatformInfoString();
-    const date = getUTCDateString();
-    if (this._heartbeatsCache?.heartbeats == null) {
-      this._heartbeatsCache = await this._heartbeatsCachePromise;
-      // If we failed to construct a heartbeats cache, then return immediately.
+      // This is the "Firebase user agent" string from the platform logger
+      // service, not the browser user agent.
+      const agent = platformLogger.getPlatformInfoString();
+      const date = getUTCDateString();
+      console.log('heartbeats', this._heartbeatsCache?.heartbeats);
```

Comment:
Is this intentional or an oversight?

Reply:
hi @hsubox76 
this is showing in our production logs now, can you please remove it
it took me a while to see where is it coming from

Reply:
After updating Firebase to version 10.13.0, I'm seeing a console log message that says `heartbeats undefined`. It would be great if the log could be removed!

Reply:
Same here

Reply:
I can't update to 10.13.0 because of this as it adds a lot of noise to my frontend monitoring logs.

Reply:
@GogoVega @robertIsaac @Tosters @BenaymS @gustavopch

This was removed in 10.13.1